docs/source/reference/_create_conda_env.md

```diff
@@ -47,12 +88,8 @@ export BLUESKY_CONDA_ENV=bluesky_2023_2
 alias become_bluesky='conda activate ${BLUESKY_CONDA_ENV}'
 ```
 
-More documentation about [conda](../reference/_conda_base.md) is available
-[elsewhere](../reference/_conda_environment.md) in this repository.
-
-TODO: how to create an alias to activate the bluesky environment (with access to APSshare vs not).
 
-## bash environment variables
+TODO: how to create an alias to activate the bluesky environment (with access to APSshare vs not) and add to FAQ.
```

Comment:
Here's what I use (assumes `conda` is available and `BLUESKY_CONDA_ENV` environment variable is defined):

```bash
alias become_bluesky='conda activate ${BLUESKY_CONDA_ENV}'
```

Reply:
Context: it's in my `~/.bash_aliases` file:

```bash
(base) jemian@otz ~/.../bluesky/environments $ grep becom ~/.bash_aliases 
alias become_bluesky='conda activate ${BLUESKY_CONDA_ENV}'
```

Reply:
I know, this needs more context  and work (how to deal with bashrc, source the bashrc_aliases etc). I can do something similar to what I wrote for the start_bluesky script tomorrow but I have to be gone in 1min.

Reply:
I found the subtitle confusing though. It is only one piece of a much bigger thing.

Reply:
give me 5 min

Reply:
Done. I remove the stuff about the broken script for now.